components/button/button-subtle-styles.js

```diff
@@ -60,15 +36,10 @@ export const buttonStyles = css`
 	}
 	button:hover,
 	button:focus,
-	:host([active]) button,
-	:host(.d2l-button-subtle-hover) button,
```

Comment:
It appears that these `d2l-button-subtle-hover` and `d2l-button-subtle-focus` classes were not needed.

Reply:
They were probably used at one point in the Galen tests, and we might want something like this for perceptual diff, but we can add it back if we do.